docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_label: '🗃 Create an NFT Collection'
+sidebar_position: 4
+---
+
+# Create an NFT Collection Using LSP8
+
+This tutorial will explore how to create a collection of unique digital assets.
+
+:::info
+⌨️ The full code of this example can be found in the 👾 [lukso-hardhat-template](https://github.com/CJ42/LUKSO-Hardhat-template).
+:::
```

Comment:
Can we stay consistent?

Why this info has no blank lines why the note below has blank line?

```suggestion
:::info

⌨️ The full code of this example can be found in the 👾 [lukso-hardhat-template](https://github.com/CJ42/LUKSO-Hardhat-template).

:::
```

Reason:
https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier